src/api/teams/team_repos.rs

```diff
@@ -0,0 +1,114 @@
+use crate::params;
+use crate::{Octocrab, Result};
+use reqwest::StatusCode;
+
+/// Handler for managing a team's repositories through
+/// GitHub's teams API.
+///
+/// Created with [`TeamHandler::repos`]
+///
+/// [`TeamHandler::repos`]: ./struct.TeamHandler.html#method.repos
+pub struct TeamRepoHandler<'octo> {
+    crab: &'octo Octocrab,
+    org: String,
+    team: String,
+}
+
+/// Whether a team manages a repository.
+pub enum ManagesRepo {
+    Yes,
+    No,
+    Err,
+}
+
+impl<'octo> TeamRepoHandler<'octo> {
+    pub(crate) fn new(crab: &'octo Octocrab, org: String, team: String) -> Self {
+        Self { crab, org, team }
+    }
+
+    /// Checks if a team manages a repository.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let manages_repo = octocrab::instance()
+    ///     .teams("owner")
+    ///     .repos("team")
+    ///     .manages("owner", "repo")
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn manages(
```

Comment:
```suggestion
    ///     .check_manages("owner", "repo")
    ///     .await?;
    /// # Ok(())
    /// # }
    /// ```
    pub async fn check_manages(
```